peekingduck/runner.py

```diff
@@ -17,12 +17,15 @@
 """
 
```

Comment:
Fixed https://github.com/aimakerspace/PeekingDuck/pull/510/commits/372a443aac630a38b54b1b6ee0904027b6344614

Checks and installs optional packages during import stage. The `RequirementChecker` class is only used during Python API mode.

Reply:
Additionally moved `LoggerSetup()` to`__init__.py` so it will be run before everything else in Python API mode too. This is necessary so that package update logging messages during the import start are properly formatted.

Reply:
This a nice fix!